### DISCUSSION TITLE
Cannot find package `@johnlindquist/kit`

### DISCUSSION THREAD

**Discussion by prplxr** [johnlindquist/kit#728] [Errors]
Hi thanks for creating kit. 
I tried to soft link a customenv `somewhereelse/kenv/kenvs/customkenv`  to `~/.kenv/kenvs`. Kit found the script fine but cannot run it. 
I am getting error `Cannot find package `@johnlindquist/kit`. 
Although scripts in ~/.kenv/scripts and other non linked scripts work fine. 

I also tried running `~/.kit/node/bin/npm install --only=prod --prefix ~/.kit` but it did not help.

Thanks

**Comment by ryuheechul** [comment]
This seems to happen when your real directory is not under (probably because of symlinking) `~/.kenv/kenvs`.

I worked around it by basically doing it below.
`ln -s ~/.kenv/node_modules node_modules` (in your `customkenv` directory)

My codified version of this is at https://github.com/ryuheechul/kenv/commit/78b935586c96fb564278913d0b2d9bb0e308ee28
